manual/glossary.md

```diff
@@ -0,0 +1,57 @@
+# Glossary
+
+## General terms
+
+**Catalogue**
+Catalogue lists all the catalogue items that users can apply for access rights.
+
+**Catalogue item**
+Catalogue items are resources the applicant can apply for access to.
+
+A catalogue item consists of a resource, a license, a workflow and the application form. For example, a biobank sample collection.
+
+**Entitlement**
+Entitlement means the access right. Entitlement is granted when the applicant has accepted the license terms and their application is approved
+
+**Resource**
+Resource means the material or the data that needs have controlled access, for example a sensitive scientific dataset. Resource is the core part of a catalogue item.
+
+**Workflow**
+A workflow determines how the application’s approval process proceeds and who is in charge of the application. Workflow is a part of a catalogue item.
+
+**Licence**
+A license defines the terms and conditions for the use of the data. In REMS, a license can be an external link, an attached file or simply text added to the application form.
+
+Licenses are connected to resources.
+
+## Roles
+
+**Applicant**
+An applicant is a user who applies for an access right to a data resource by filling in the application form.
+
+**Member**
+A member is a user who has been added to an application by an applicant or a handler. Members are often part of the applicant’s research group.
+
+Members will be granted access rights once the application is approved and they have accepted the licenses.
+
+**Handler**
+A handler is a user who processes the access rights applications. Generally, handlers decide whether the application should be approved or rejected. They can also request reviews and decisions from other users.
+
+Resource owners assign handlers.
+
+**Decider**
+Decider is a user who can approve or reject applications suggested by handlers. Decider role is used when a handler should not be able to approve or reject applications, for example in governmental office use.
+
+**Owner**  
+An owner is a user who lists and manages their data resources in REMS. Owners can create organisations and assign organisation owners. Owners also assign handlers and reviewers to process the applications.
+
+Owner role is always assigned by the REMS admins.
+
+**Reporter**
+Reporter is a user who can view all applications. Reporters are only available when using REMS API.
```

Comment:
Reporter can use also the UI to view the applications.

Reply:
Changed accordingly.